easybuild/easyconfigs/s/supermagic/supermagic-20170824-foss-2017a.eb

```diff
@@ -0,0 +1,32 @@
+##
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'supermagic'
+version = '20170824'
+
+homepage = 'https://github.com/hpc/supermagic'
+description = """Very simple MPI sanity code. Nothing more, nothing less."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+git_commit_id = '07311ad853'
+sources = ['%s.tar.gz' % git_commit_id]
+source_urls = ['https://github.com/hpc/supermagic/archive']
```

Comment:
@jordiblasco We have a better way of handling this now, i.e. to rename the sources on download:

```python
sources = [{'download_filename': '%s.tar.gz' % git_commit_id, 'filename': SOURCE_TAR_GZ,
```

(see also http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources)

Reply:
Also, please include a SHA256 checksum:

```python
checksums = ['...']
```